### PR TITLE
Renamed all references of insight to signal

### PR DESCRIPTION
**Pull request by onursumer**
- API end point is still https://www.genomenexus.org/insight/, that should be fixed on the Genome Nexus side.
- We should probably rename the repository to `signal` or `signal-db` as well.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) MSK-INSIGHT
+Copyright (c) SIGNAL-DB
```
Comment (onursumer) [inline LICENSE]: Should it be just `SignalLDB` instead?

File: src/pages/Gene.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ class Gene extends React.Component<IGeneProps>
     }
 
     private get loader() {
-        return loaderWithText("Fetching INSIGHT(ful) alterations...");
+        return loaderWithText("Fetching SIGNAL(ing) alterations...");
```
Comment (onursumer) [inline src/pages/Gene.tsx]: SIGNAL(ing) does not sound as good as INSIGHT(ful)

Comment (inodb) [reply src/pages/Gene.tsx]: lol yeah prolly better to change to just `fetching alterations`, unless you have a better pun for signaldb

File: LICENSE (hunk 1 above)
Comment (inodb) [reply LICENSE]: hmm it all looks funky lol, too many CAPS - `SIGNALDB`?

**Pull request merged**